prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -68,7 +70,9 @@ class CustomFHIRFunctionsTests {
         // Individual function results are tested on their own unit tests.
         CustomFHIRFunctions.CustomFHIRFunctionNames.values().forEach {
             // todo: this is temporary until this code is moved
-            if (it != CustomFHIRFunctions.CustomFHIRFunctionNames.LivdTableLookup) {
+            if (it != CustomFHIRFunctions.CustomFHIRFunctionNames.LivdTableLookup &&
+                it != CustomFHIRFunctions.CustomFHIRFunctionNames.ChangeTimezone
```

Comment:
Remove ChangeTimezone from this if. We removed Livd due to its dependency on metadata

Reply:
Still needed to handle this specially because the generic inputs used in this `forEach` would cause `changeTimezone` to fail, but instead of skipping it altogether, I just added a specific `if` block for that case.